documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -110,6 +110,18 @@ We instead read from the _right_, accounting for the number of trusted proxies.
 
 The maximum request body size to accept in bytes including while streaming. Defaults to 512kb. You can disable this option with a value of 0 and implement a custom check in [`handle`](hooks#server-hooks-handle) if you need something more advanced.
 
+### `CERT_PATH`, `CERT_KEY_PATH`, and `HTTPS_PORT`:
+By default, the server only listens for plain HTTP requests. To enable an additional HTTPS endpoint, you must provide the `CERT_PATH` and `CERT_KEY_PATH` environment variables pointing to a file containing a valid certificate and a file containing its associated private key respectively, both in the PEM format.
+The files are read once on startup, so if the contents of the files change, the app will need to be restarted.
+
+The default port that will be used for the HTTPS endpoint is 3001. You can customize this via the `HTTPS_PORT` environment variable.
+
```

Comment:
```suggestion

> We recommend using a reverse proxy instead and only use these options when you have no other possibilities of securing the connection

```

Reply:
Added a slightly reworded version of this in 083af9a69237bb6d8a4df00a6d2b1c6c8767961a
Also, I just noticed that variables like `export const cert_path = env('CERT_PATH', false);` are actually exported, so according to the [style guide](https://github.com/sveltejs/kit/blob/master/CONTRIBUTING.md#coding-style) they should camelCaes, am I right?

> Internal variables are written with snake_case while external APIs are written with camelCase

Do these count as "external APIs"?

Reply:
I think they should be camel case here yes, good catch

Reply:
personally I think the recommendation should be in the opposite direction. this seems better than a reverse proxy to me as I explained here: https://github.com/sveltejs/kit/pull/10183#issuecomment-1614066698

perhaps we should just remove this section if we can't agree on a recommendation?